pages/_app.tsx

```diff
@@ -112,15 +127,21 @@ const App: FC<AppProps> = (props) => {
 				<meta name="twitter:url" content="https://kwenta.eth.limo" />
 				<link rel="icon" href="/images/favicon.svg" />
 			</Head>
-			<RecoilRoot>
-				<QueryClientProvider client={new QueryClient()}>
-					<WagmiConfig client={wagmiClient}>
-						<WithAppContainers>
-							<InnerApp {...props} />
-						</WithAppContainers>
-					</WagmiConfig>
-				</QueryClientProvider>
-			</RecoilRoot>
+			<Provider store={store}>
+				{/* <SDKContext.Provider
```

Comment:
Yea I think like you identified, we don't really need the sdk context as can reference it directly due to being a singleton